content/sidebar/bseu_history.md

```diff
@@ -0,0 +1,8 @@
+title: О БГЭУ
+slug: links_applicant
```

Comment:
`slug` should be unique across the site. Something like `bseu_history` would suit.